auth.ts

```diff
@@ -48,7 +49,7 @@ export const {
       }
 
       if (account?.provider === "google" && profile?.email && profile?.email_verified) {
-        return profile.email_verified && (profile.email.endsWith("@nuclaysolutions.com") || profile.email.endsWith("@givecentral.org") || profile.email.endsWith("@terminal33.io"))
```

Comment:
Why are the conditions removed? If you dont want to use them, put them with prod env check

Reply:
doing it